collectors/30/kafka_offset_collector.py

```diff
@@ -40,60 +45,78 @@ def format_kafka_consumer_offset_tsd_key(consumer_group, topic, partition, offse
 # KAFKA OFFSET MONITORING
 
 
-def get_partitions(zk, group, topic):
-    return zk.get_children(KafkaPaths.topic_offsets(group, topic))
+def get_partitions(zk, topic):
+    path = KafkaPaths.partitions(topic)
+    return map(int, zk.get_children(path))
+
+
+def get_kafka_topics(zk):
+    path = KafkaPaths.topics()
+    return [str(topic) for topic in zk.get_children(path) if topic not in BLACKLIST]
```

Comment:
Do you want to implement the blacklist via regex so maybe topics that end or start with "-test" are excluded? That way we don't have to deploy for each test topic.